### PR TITLE
Added test for expired JWT

### PR DESCRIPTION
**Pull request by honza-kasik**
Please make sure your PR meets the following requirements:
- [x] Pull Request contains a description of the changes
- [x] Pull Request does not include fixes for multiple issues/topics
- [x] Code is formatted, imports ordered, code compiles and tests are passing
- [ ]  N/A Link to the passing job is provided
- [x] Code is self-descriptive and/or documented
- [x] Description of the tests scenarios is included (see #46)

### CODE REVIEW

File: microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/compatibility/ExpiredJwtTest.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                .jwtId(UUID.randomUUID().toString())
+                .audience("microprofile-jwt-testsuite")
+                .subject(subject)
+                .userPrincipalName(subject)
+                .issuer("issuer")
+                .issuedAtTime(now.getEpochSecond())
+                .expirationTime(sooner.getEpochSecond())
+                .groups(Collections.emptySet())
+                .customClaim("preferred_username", subject)
+                .build());
```
Comment (fabiobrz) [inline microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/compatibility/ExpiredJwtTest.java]: I am wondering whether the literals that you're using to build the claims here should be defined as constants somewhere, as I see they're actually used in `jwt-auth-tool` module, mainly. WDYT?

Comment (honza-kasik) [reply microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/compatibility/ExpiredJwtTest.java]: Thanks for catching this. The custom claim setter can be removed. There is however issue with issuer which should be the same value for each test, so I will think on how to implement this as a constant...

Comment (honza-kasik) [reply microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/compatibility/ExpiredJwtTest.java]: Solving in #171  ...

Comment (honza-kasik) [reply microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/compatibility/ExpiredJwtTest.java]: Solved.

**Pull request merged**